gpflow/optimizers/mcmc.py

```diff
@@ -109,13 +103,13 @@ def grad_fn(dy, variables: Optional[tf.Tensor] = None):
 
     def convert_constrained_values(self, hmc_samples):
```

Comment:
the current name sounds like you need to pass in constrained values, how about
```suggestion
    def convert_to_constrained_values(self, hmc_samples):
```
instead?